app.py

```diff
@@ -87,6 +98,35 @@ def events(account_id):
     dripData.events.append(request.json["events"][0])
     return Response(None, status=204)
 
+@app.route("/v2/<account_id>/tags", methods=["GET", "POST"])
+def tags(account_id):
+    if request.method == "GET":
+        return {
+            "tags" : dripData.tags
+        }
+    elif request.method == "POST":
+        data = request.json.get("tags", [])[0]
```

Comment:
This will throw IndexError if there aren't any tags.

Reply:
Fixed